extensions/prototype-content-with-issues-table/app.R

```diff
@@ -98,28 +68,60 @@ server <- function(input, output, session) {
   # initialize Connect API client
   client <- connect()
   
-  # get content once up front and pass it around for additional filtering
-  content <- get_content(client, limit = inf)
-  
-  # cache content list
+  # TODO: use `v1/content/failed` when #30414 merges so we only list content we
+  # know has failed before, filter to deployed within last 60d for now
```

Comment:
As discussed in the Metrics sync today we needed to introduce temporary date range restrictions to improve load times on connect.posit.it. Until we surface failed content job data via `v1/content/failed` we are filtering to content deployed within the last 60 days. Additionally, since completed jobs are reaped every 30 days by default we only collect usage data for visits within that last 30 days.